README.md

```diff
@@ -32,28 +34,71 @@ Google Earth Engine requires a **Google Cloud Project**. To find your Project ID
 - Select the sign-in under `Plugins > Google Earth Engine > Sign-In`.
 - Follow prompts to enter your project ID.
 
-### 4️⃣ Use Earth Engine Directly in QGIS 🎉
+---
+
+## 🌍 Using Earth Engine in QGIS
 
-Once the plugin is set up, you can start using Earth Engine within QGIS! Try adding a dataset:
+### 1. Code Usage
+
+You can use Earth Engine datasets programmatically within QGIS using the `Python Console`:
 
 ```python
 import ee
 from ee_plugin import Map
 
-# Load an elevation dataset
 image = ee.Image('USGS/SRTMGL1_003')
-
-# Define visualization parameters
 vis_params = {
     'min': 0, 'max': 4000,
     'palette': ['006633', 'E5FFCC', '662A00', 'D8D8D8', 'F5F5F5']
 }
-
-# Add the image layer to QGIS
 Map.addLayer(image, vis_params, 'Digital Elevation Model')
 Map.setCenter(-121.753, 46.855, 9)
 ```
 
+### 2. Algorithms via Processing Toolbox and Plugin Menu
+
+Earth Engine algorithms are available from:
+
+- The **Processing Toolbox** (`Processing > Toolbox > Google Earth Engine`)
+- The **Plugin Menu** (`Plugins > Google Earth Engine`)
+
+See [available algorithms](#-available-algorithms) for more details.
+
+![example algorithm](./media/example_algorithm.png)
+
+### 3. Model Designer
+
+Earth Engine algorithms can be integrated into QGIS **Model Designer** workflows:
+
+- Open `Processing > Graphical Modeler`
+- Add and chain EE algorithms with other QGIS algorithms
+- Save your custom processing model for repeated use
+
+An [example model](./examples/srtm_hillshade.model3) is provided for the hillshade example below:
```

Comment:
Not sure that this link will work.  I think it needs to be like this for it to work from Github:
```suggestion
An [example model](./blob/master/examples/srtm_hillshade.model3) is provided for the hillshade example below:
```

However, for the website, we may need to do something like this:
```suggestion
An [example model](https://github.com/gee-community/qgis-earthengine-plugin/blob/master/examples/srtm_hillshade.model3) is provided for the hillshade example below:
```